[2022-12-06T20:55:36.469+0000] {taskinstance.py:1159} INFO - Dependencies not met for <TaskInstance: dag.pull_data manual__2022-12-06T20:55:32.165103+00:00 [queued]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_tasks_state={'total': 1, 'successes': 0, 'skipped': 0, 'failed': 0, 'removed': 0, 'upstream_failed': 0, 'done': 0}, upstream_task_ids={'create_table'}
[2022-12-06T20:55:36.482+0000] {local_task_job.py:98} INFO - Task is not able to be run
